src/index.ts

```diff
@@ -1,4 +1,4 @@
 import Client from './client';
 import { LocalStream, RemoteStream, Constraints } from './stream';
-import { Signal, Trickle, IonSFUJSONRPCSignal } from './signal';
-export { Client, LocalStream, RemoteStream, Constraints, Signal, Trickle, IonSFUJSONRPCSignal };
+import { Signal, Trickle, IonSFUJSONRPCSignal, IonSFUGRPCGWebSignal } from './signal';
```

Comment:
we should remove signaling modules from export. otherwise when someone uses the package they will bundle deps from all signal modules

Reply:
If we do not export, do we need to create a Factory class?

Reply:
@tarrencev any good ideas?

Reply:
@tarrencev @adwpc I added interface create functions to replace the object export.

```ts
import { Signal, Trickle, IonSFUJSONRPCSignal, IonSFUGRPCWebSignal } from './signal';
export { Client, LocalStream, RemoteStream, Constraints, Signal, Trickle };

export function createJSONRPCSignal(uri: string): Signal {
    return new IonSFUJSONRPCSignal(uri);
}

export function createGRPCWebSignal(uri: string): Signal {
    return new IonSFUGRPCWebSignal(uri);
}
```